include/psa/crypto.h

```diff
@@ -2495,6 +2495,14 @@ psa_status_t psa_aead_update_ad(psa_aead_operation_t *operation,
  *          - In particular, do not copy the output anywhere but to a
  *            memory or storage space that you have exclusive access to.
  *
+ * This function does not require the input to be aligned to any
+ * particular block boundary. If the implementation can only process
+ * a whole block at a time, it must store the last partial input block
+ * or adjust its internal state accordingly until the next call to
+ * psa_aead_update(), psa_aead_finish() or psa_aead_verify(), and produce
+ * the corresponding output when sufficient input is available or on the
+ * finish or verify call.
```

Comment:
Is this asking too much of implementations (for accelerators or secure elements)?

Reply:
The point of this PR is precisely to avoid asking too much. But the wording could use some improvements. What did you think this is asking?

Reply:
I've reworded this paragraph.